README.md

```diff
@@ -1,13 +1,25 @@
 Ad Hoc homework problems
 ========================
 
-Each directory contains a homework problem, four in total. Please complete one--or more--of the four before submitting. Make sure to read the README in
-each directory, it will give you important information on completing each problem.
+Each directory contains a homework problem, four in total. Please complete one--or
+more--of the four before submitting. Make sure to read the README in each directory,
+it will give you important information on completing each problem.
 
 We're on the honor system here. Don't get help from, or give help to, another
 person or a resource on the internet. Do the work yourself. It's okay if you
 need to consult language docs or the MDN or similar.
 
+Which problems should I try?
+------------
+
+Well, that depends on the position you're applying for! Here are some suggestions:
+
+ - Front-end Developer: [hhbuilder](https://github.com/adhocteam/homework/tree/master/hhbuilder)
```

Comment:
Nice!  I also agree that some guidance for candidates is a good thing.  

However, by not recommending `slcsp` to a position, I think candidates will not attempt it.  `slcsp` has always been the one I have the most trouble grading so I wouldn't necessarily mind.